### ISSUE TITLE
About training loss curve: is this normal?

### ISSUE THREAD

**Issue by yuxuanzhou97** [GauleeJX/flodiff#9]
Hello, thank you very much for your excellent work! 

While trying to train the network myself, I found that the loss behaves like this—does this phenomenon seem abnormal?

<img width="2485" height="1099" alt="Image" src="https://github.com/user-attachments/assets/577a89de-50a8-4695-884f-6c95a87a5c4a" />

**Comment by wikiahuang** [comment]
Yes, that’s expected since a 4-epoch warm-up is applied.
